src/main/resources/stddef.dm

```diff
@@ -0,0 +1,344 @@
+// Copyright © 2016 BYOND Software.
+
+// This file is not covered under the normal GPL license.
```

Comment:
There is no need for these comments, just make note of it in the `README` and `LICENSE` files and keep the `// Copyright © 2016 BYOND Software.`